consensus/ibft/src/test/java/tech/pegasys/pantheon/consensus/ibft/validation/NewRoundMessageValidatorTest.java

```diff
@@ -147,16 +145,6 @@ public void newRoundTargetingDifferentSequenceNumberFails() {
     assertThat(validator.validateNewRoundMessage(inValidMsg)).isFalse();
   }
 
-  @Test
```

Comment:
isn't this test still valid?

Reply:
The mocked messageValidatorFactory (and MessageValidator) ultimately means it doesn't matter WHY a message is invalid - just that it is - thus many of the specialised failure modes can be removed.